src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let trueRobot = {};
+
+  for (const key in robot) {
+    if (trueRobot[robot[key]]) {
+      trueRobot = null;
+
+      return trueRobot;
+    } else {
+      trueRobot[robot[key]] = key;
```

Comment:
if you return `null` function will be over and no need to add `else`; just put this line after the condition